### DISCUSSION TITLE
0033. 搜索旋转排序数组

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#147] [Comments]
# 0033. 搜索旋转排序数组 | 算法通关手册

[https://algo.itcharge.cn/solutions/0001-0099/search-in-rotated-sorted-array/](https://algo.itcharge.cn/solutions/0001-0099/search-in-rotated-sorted-array/)

**Comment by xiangxichen26** [comment]
有笔误：如果  mid[mid]==target，说明找到了 target，直接返回下标。 应该是nums[mid]

**Reply by itcharge** [reply]
感谢指正，文中已更新。
